core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -103,7 +110,14 @@ class LoadBalancerService(config: WhiskConfig, verbosity: LogLevel)(
      *
      * @return a map where the key is the subject and the long is total issued activations by that user
      */
-    override def getUserActivationCounts: Map[String, Long] = userActivationCounter.toMap mapValues { _.cur }
+    override def getIssuedUserActivationCounts: Map[String, Long] = userActivationCounter.toMap mapValues { _.cur }
+
+    /**
+     * Retrieves a per subject map of counts representing in-flight activations as seen by the load balancer
+     *
+     * @return a map where the key is the subject and the long is total issued activations by that user
+     */
```

Comment:
Those method will be eliminated or redone on the next PR (dead code and remove consul from invoker side).